types/office-js-preview/index.d.ts

```diff
@@ -10541,7 +10541,7 @@ declare namespace Office {
     /**
      * Represents an attachment on an item from the server. Read mode only.
      *
-     * An array of AttachmentDetail objects is returned as the attachments property of an Appointment or Message object.
+     * An array of `AttachmentDetails` objects is returned as the attachments property of an appointment or message item.
```

Comment:
This is tangential, but we should figure out when we use code ticks and when we don't. When is it helpful and when is it cluttered (especially for Intellisense considerations)?

Reply:
I'll start with the MS Style Guide and go from there.

Reply:
I'll make a broader update in a subsequent PR. Guidance seems to be that names of objects/classes, properties/members, and methods/functions are bold, while code samples should be monospaced (which I read as using ``).